node/src/types/deploy.rs

```diff
@@ -1145,6 +1145,23 @@ impl Deploy {
         Self::random_transfer_with_session(rng, session)
     }
 
+    pub(crate) fn random_expired_deploy(rng: &mut TestRng) -> Self {
```

Comment:
I don't think there's a need to use randomized deploy generator for this particular test, especially that what we're testing - TTL expiration - is hardcoded.

Reply:
Yes, but its just really trying to keep in step with the convention thats emerged around tests related for a DA.

Reply:
We don't have to follow the convention if it's not helpful but only adds complexity.

Reply:
I prefer using random (reproducible) data for unit tests like this, since it occasionally catches bugs which hard-coded values wouldn't.

Reply:
I meant that the test we're adding here is only verifying that a too small TTL will be rejected and this particular property of the "random" deploys is hardcoded.

Reply:
Yup - but aside from liking the randomness (even though the particular part being tested isn't random), I think we'd just be moving the complexity to the test, since it's not geared up at all just now to create a "real" deploy.

Reply:
What I was suggesting was to remove the `random_` from the method name and the `rng` argument and just have a deploy that is already expired and use that in a test. Currently, this method might be giving a false impression of testing corner cases, multiple possible way a deploy can expire, etc. while in reality is not.